framework/tlvf/yaml/tlvf/WSC/WSC_Attributes.yaml

```diff
@@ -265,6 +265,18 @@ sWscAttrVersion2:
     _type: uint8_t
     _value: WSC_VERSION2
 
+sWscAttrVersion2:
```

Comment:
It's not an attribute, it's a vendor extension IE. So `sWscIeVersion2` would be better. (nitpick)

With that, it is also no longer needed to change the name of sWscAttrVersion2.